core/src/main/java/io/seata/core/store/db/AbstractDataSourceProvider.java

```diff
@@ -195,6 +202,18 @@ protected String getUser() {
      */
     protected String getPassword() {
         String password = CONFIG.getConfig(ConfigurationKeys.STORE_DB_PASSWORD);
+        String publicKey = getPublicKey();
+        if (StringUtils.isNotBlank(publicKey)) {
+            try {
+                password = ConfigTools.decrypt(publicKey, password);
+            } catch (Exception e) {
+                if (LOGGER.isErrorEnabled()) {
```

Comment:
The error log does not need to add this judgment.

Reply:
ok